src/command/dev/mod.rs

```diff
@@ -86,6 +92,19 @@ pub struct SupergraphOpts {
         conflicts_with_all = ["subgraph_name", "subgraph_url", "subgraph_schema_path"]
     )]
     supergraph_config_path: Option<Utf8PathBuf>,
+
+    /// A graph ref that is accessible in Apollo Studio.
```

Comment:
`graphRef`? or, maybe `graph reference`? I don't know how common `graph ref` is outside of apollo